src/components/Dashboard/FilingHistoryList.vue

```diff
@@ -645,28 +645,17 @@ export default class FilingHistoryList extends Mixins(
     if (index >= 0) this.togglePanel(index, this.historyItems[index])
   }
 
-  private startCorrectionDialog (item: HistoryItemIF): void {
-    this.currentFiling = item
-    this.fileCorrectionDialog = true
-  }
```

Comment:
This is small and was only called from 1 place, so I moved it there instead.

Reply:
This will be used to start all types of correction filings later.

Reply:
Yes. At that time, we can clean up `correctThisFiling()` since it will be the same path for all corrections.

Reply:
For sure.